impl/src/bazel.rs

```diff
@@ -23,6 +23,35 @@ use crate::{
   util::RazeError,
 };
 
+use std::{
+  env,
+  path::{Path, PathBuf},
+};
+
+/** Returns a path to a Bazel workspace root if one could be found, otherwise None */
+pub fn find_workspace_root(current_dir: Option<&Path>) -> Option<PathBuf> {
+  let default = env::current_dir().unwrap();
```

Comment:
In my comment on lines 47-52, I mentioned trying to eliminate the input parameter.

Perhaps, instead of eliminating the parameter, you could have the user of the function (`main`) pass this in.  That would make the behavior more clear, and also let you handle an `Err` from `current_dir()`, rather than calling unwrap.

Reply:
I've updated `find_workspace_root` to handle errors and not panic anymore.